src/app/components/aboutPage/aboutPage.component.tsx

```diff
@@ -2,104 +2,94 @@ import React from 'react';
 
 import './aboutPage.component.scss';
 import rocket from '../../images/rocket.svg';
+import teamMember from '../../images/teamMember.svg';
 
 export function AboutPage() {
     return (
         <div className="about-page-items">
             <h2>About page</h2>
 
-            <div className="about-page-item">
-                <div className="about-page-item-title">
-                    <img alt="White rocket that moves on hover" src={rocket} />
-                    <h3>Some facts</h3>
-                </div>
-                <div className="about-page-list">
-                    <ul>
-                        <li>There are over 140 spaceflight launches in 2021</li>
-                        <li>
-                            There are at least 7 people on space station right
-                            now
-                        </li>
-                        <li>
-                            In 24 hours, the space station is travelling through
-                            16 sunrises and sunsets
-                        </li>
-                        <li>
-                            The space for living and working on space station is
-                            larger than a six-bedroom house
-                        </li>
-                    </ul>
-                    <p>
-                        <em>
-                            Source:
-                            <br />
-                            <a
-                                href="https://www.nasa.gov/feature/facts-and-figures"
-                                target="_blank"
-                            >
-                                International Space Station Facts and Figures
-                            </a>
-                        </em>
-                    </p>
-                </div>
+            <div className="about-page-item-title">
+                <img alt="White rocket that moves on hover" src={rocket} />
+                <h3>Some reasons to keep on track with spaceflights</h3>
             </div>
+            <ul className="about-page-list">
+                <li>
+                    <a
+                        href="https://www.nasa.gov/feature/facts-and-figures"
+                        target="_blank"
+                    >
+                        There are over 140 spaceflight launches in 2021
```

Comment:
not in scope of this PR, but it's good example where you could try to utilise config driven data